### PR TITLE
[ERC20 Launchpad] - UI for Launch token community step

### PR DESCRIPTION
**Pull request by mzparacha**
## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8706

## Description of Changes
- Added UI for Launch token community step of tokenized community creation flow

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_TOKENIZED_COMMUNITY`
- Visit `/createTokenCommunity`
- Fill the form - verify the form validation works as expected
- Submit
- Verify you get redirected to step 3 "Sign and Launch" with a "Not Implemented" label
- Verify there are no regressions with the normal community creation flow

## Deployment Plan
N/A

## Other Considerations
N/A

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/CommunityInformationForm/CommunityInformationForm.tsx [hunk 1]
```diff
@@ -265,7 +196,7 @@ const BasicInformationForm = ({
         enableGenerativeAI
       />
 
-      {socialLinksDisplay ? (
+      {withSocialLinks ? (
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/CommunityInformationForm/CommunityInformationForm.tsx:199]: this comment is not to this line but to image upload. According to the ticket, image shuold be optional but so far we require it in community creation flow. My point is that we should make the same validation for both flows

**Review by masvelio — APPROVED** [review]
lgtm

File: packages/commonwealth/client/scripts/views/components/CommunityInformationForm/CommunityInformationForm.tsx (hunk 1 above)
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/components/CommunityInformationForm/CommunityInformationForm.tsx:199]: yeah its actually required (we have no community without an image) - so I kept it required.

File: packages/commonwealth/client/scripts/views/components/component_kit/cw_cover_image_uploader.tsx [hunk 2]
```diff
@@ -228,7 +228,10 @@ export const CWCoverImageUploader = ({
 
     setIsUploading(true);
 
-    const [_imageURL, _uploadStatus] = await uploadImage(file);
+    // const [_imageURL, _uploadStatus] = await uploadImage(file);
```
[the remark this answers is not in the document]
Comment (mzparacha) [reply packages/commonwealth/client/scripts/views/components/component_kit/cw_cover_image_uploader.tsx:231]: done

**Review by burtonator — APPROVED** [review]

**Pull request merged**